neuro_evolution_ctrnn/optimizer/i_optimizer.py

```diff
@@ -140,3 +123,116 @@ def shape_fitness_weighted_ranks(self, population):
             if self.conf.efficiency_weight:
                 shaped_fitness += self.conf.efficiency_weight * ind.efficiency_rank
             ind.fitness.values = tuple([shaped_fitness])
+
+    def evaluate_candidates(self, candidates):
+        if self.initial_seed:
+            seed_for_generation = None
+            if self.conf.fix_seed == IOptimizer.FIX_SEED_FALSE:
+                seeds_for_evaluation: np.ndarray = np.random.randint(1, 10000, size=len(candidates))
+                seeds_for_recorded: np.ndarray = np.random.randint(1, 10000,
+                                                                   size=len(self.recorded_individuals))
+            elif self.conf.fix_seed == IOptimizer.FIX_SEED_PER_GENERATION:
+                seed_for_generation = np.random.randint(1, 10000)
+                seeds_for_evaluation: np.ndarray = seed_for_generation * np.ones(len(candidates), dtype=np.int64)
+                seeds_for_recorded = seed_for_generation * np.ones(len(self.recorded_individuals),
+                                                                   dtype=np.int64)
+            elif self.conf.fix_seed == IOptimizer.FIX_SEED_GLOBAL:
+                seed_for_generation = self.initial_seed
+                seeds_for_evaluation: np.ndarray = seed_for_generation * np.ones(len(candidates), dtype=np.int64)
+                seeds_for_recorded = seed_for_generation * np.ones(len(self.recorded_individuals),
+                                                                   dtype=np.int64)
+            else:
+                raise NotImplementedError()
+        else:
+            seed_for_generation = 0
+            seeds_for_evaluation = np.zeros(len(candidates), dtype=np.int64)
+            seeds_for_recorded = np.zeros(len(self.recorded_individuals), dtype=np.int64)
+
+        brain_genomes_recorded = self.strip_strategy_from_population(self.recorded_individuals)
+        nevals = len(candidates) + len(brain_genomes_recorded)
+
+        novelty_data = (
```

Comment:
alles was mit novelty zu tun hat, sollte mMn ersatzlos entfernt werden. Also insb. `seeds_for_recorded`, `brain_genomes_recorded`, `novelty_data` und `self.conf.novelty`. 

(Sollte jedoch in einem Separatem PR geschehen. Würde ich "demnächst" auh selbst machen)